sip-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -51,6 +51,10 @@ under the License.
             <validationRegex>^(?!.*([._%+-])\1)[a-z0-9\._]+$
             </validationRegex>
         </requiredProperty>
+        <requiredProperty key="useLombok">
```

Comment:
Default value is missing.

Reply:
Using validation and defaultValue at the same time within requiredProperty is not possible. If defaultValue is used, or if value for the property is provided with maven command as a parameter, in those cases validation is ignored.

Reply:
In that case I would sacrifice validation. If we offer 'y' as default value, and we check for the same value in dependency condition validation is not that important. it can be y or something else which suggests no.